### ISSUE TITLE
Nature Methods Rebuttal

### ISSUE THREAD

**Issue by mdhaber** [scipy/scipy-articles#247]
We need a letter responding to reviewer comments. I'll get it started and submit a PR.

**Comment by ilayn** [comment]
Sorry for asking the obvious but where can we read the review letter?

**Comment by mdhaber** [comment]
We decided that the complete reviews should probably not be posted publicly, so currently only the point-by-point comments are posted (as individual issues). @rgommers @tylerjereddy @stefanv should we send the complete email to scipy-core-dev-team?

**Comment by ilayn** [comment]
I don't want this to come out passive aggressive or angry. I mean quite positive; doesn't that imply that you can also do the response letter internally as corresponding authors? We can help out with the missing ingredients but since we lack the context I don't think you need to work it out here. 

I mean you can but I guess I'll leave it alone so you can work it out :)

**Comment by mdhaber** [comment]
We'll still do everything here on GitHub, as we appreciate your feedback on whatever you have enough context to judge. You can also ignore things if you think important information is missing. You can certainly ignore this issue, as I've already assigned myself to it.

Again, all the point-by-point comments that we need to respond to are public (#222 - #231).  All the relevant content from the reviewers is repeated in those verbatim, and depending on what they others think, I'll send the full letter to the core-dev list.

**Comment by tylerjereddy** [comment]
Core dev email list should be fine for the reviewer comments--they are all authors right?!

There's usually some boilerplate at the start of the cover letter thanking reviewers for nice things they may have said--we may be better off opting to put that in "privately" after the main draft of the letter is done in public.

`latexdiff` for a highlighted version of the revised ms. sounded like a good idea to me (?) along with the letter. I was waiting on the last two PRs / comments to be finalized (they've been kind of sitting without much activity), but happy for you to take the lead :)

**Comment by rgommers** [comment]
+1 for forwarding the reviewer/editor feedback to the core team

**Comment by mdhaber** [comment]
@tylerjereddy 
> I was waiting on the last two PRs / comments to be finalized (they've been kind of sitting without much activity

Would be great if you're still willing! I thought of this as a separate task; please see #249. Also, contributions to #248 are welcome; just wanted to get it started. I'll post what I intend to work on there so we don't duplicate effort.
